src/schema/v2/marketingCollections.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  GraphQLBoolean,
+  GraphQLFieldConfig,
+  GraphQLList,
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLInt,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+import { pageable } from "relay-cursor-paging"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { connectionFromArraySlice } from "graphql-relay"
+import { MarketingCollectionsSorts } from "./sorts/marketingCollectionsSort"
+import { connectionWithCursorInfo } from "schema/v2/fields/pagination"
+import { IDFields, NodeInterface } from "./object_identification"
+
+const MarketingCollectionType = new GraphQLObjectType<any, ResolverContext>({
+  name: "MarketingCollectionType",
+  interfaces: [NodeInterface],
+  fields: {
+    ...IDFields,
+    slug: {
+      type: GraphQLString,
```

Comment:
not sure if all these types are nullable, I think we need to use `GraphQLNonNull` for the fields that are never null

Reply:
Thanks for the reminder! I was about to add that:)